requirements.txt

```diff
@@ -24,3 +24,5 @@ llama-index-storage-docstore-postgres==0.1.3
 llama-index-storage-docstore-redis==0.1.2
 llama-index-vector-stores-postgres==0.1.7
 redis===5.0.4
+git+https://github.com/polux0/Wikipedia
+llama-index-readers-wikipedia===0.1.4
```

Comment:
If we're not using the WikipediaReader in `llama-index` package maybe removing it would be better.